### PR TITLE
To Do list: interactive 

### PR DESCRIPTION
**Pull request by denisdiaconu**
i added some functionality to the application to make it interactive

### CODE REVIEW

**Comment by denisdiaconu** [comment]
Hello
I had to create a new pull req, because my main branch had a bug that didn't let my least code reviewer review it.

File: src/index.js [hunk 1]
```diff
@@ -1,33 +1,89 @@
...
+};
+const drophandler = ({ i }) => {
+  const list = handleGetData();
+  const tmp = list[selectedIndex];
+  list[selectedIndex] = list[i];
+  list[i] = tmp;
+  localStorage.setItem('todo', JSON.stringify(list));
+  window.location.reload();
+};
+const dragstarthandler = () => {
```
Comment (DeliceLydia) [inline src/index.js]: - [ ] As the requirements mentioned the project is supposed to have 2 new files imported as modules
  - One should contain methods related to drag / drop and sorting functionality.
  - The other should contain methods related to the status updates (`completed`: `true` / `false`). 
  Please work on that!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @denisdiaconu , 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denisdiaconu** [comment]
I imported 2 new files as modules.
One is having the methods of 'drag/drop'
One is having the methods of 'status update for checkbox'

**Review by carlos-ssh — APPROVED** [review]
Hi @denisdiaconu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**